src/libs/actions/BankAccounts.ts

```diff
@@ -398,24 +335,23 @@ function verifyIdentityForBankAccount(bankAccountID, onfidoData) {
 }
 
 function openWorkspaceView() {
-    API.read('OpenWorkspaceView');
+    API.read('OpenWorkspaceView', {}, {});
 }
 
-function handlePlaidError(bankAccountID, error, error_description, plaidRequestID) {
+function handlePlaidError(bankAccountID: number, error: string, errorDescription: string, plaidRequestID: string) {
     API.write('BankAccount_HandlePlaidError', {
         bankAccountID,
         error,
-        error_description,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        error_description: errorDescription,
```

Comment:
Maybe we should ask Expensify to fix this field on the backend like we did in other PR?

Reply:
@cead22 Could you please update the backend to accept both `error_description` and `errorDescription`  for this API endpoint so we can get rid of the naming convention error?

Reply:
Yes, I can submit a PR for that

Reply:
@cead22 Thank you!

Reply:
PR is merged and should be in prod on Tuesday